### PR TITLE
Add composer require to sulu upgrade docs

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Fixed tickets | fixes #issuenum
| Related PRs | sulu/sulu#prnum
| License | MIT

#### What's in this PR?

Add composer require to sulu upgrade docs.

#### Why?

Avoids that a lower sulu version is installed then a developer would expect.

### CODE REVIEW

File: upgrades/upgrade-2.x.rst [hunk 1]
```diff
@@ -5,52 +5,67 @@ This upgrade guide describes how to upgrade a Sulu 2.x project to any newer vers
...
+1. Update the sulu/sulu package
+-------------------------------
 
-   The ``sulu/sulu`` package implement the functionality of the Sulu content management system. Execute the following
-   command in the root folder of your project to update it to the newest version:
+The ``sulu/sulu`` package implement the functionality of the Sulu content management system. Execute the following
+command in the root folder of your project to update it to the version you want.
+
+First you should specify to which version you want to upgrade to, by running the following command
+by replacing ``~x.x.x`` with a valid version string like ``~2.2.7``.
```
Comment (niklasnatter) [inline upgrades/upgrade-2.x.rst]:
```suggestion
The ``sulu/sulu`` package implements the functionality of the Sulu content management system. To update this package, you need to update the version constraint for the package in the ``composer.json`` of your project. 

To do this, you can replace the ``~x.x.x`` with the a version constraint like ``~2.2.7`` and execute the following
command in the root folder of your project:
```

File: upgrades/upgrade-2.x.rst [hunk 2]
```diff
@@ -5,52 +5,67 @@ This upgrade guide describes how to upgrade a Sulu 2.x project to any newer vers
...
+
+.. code-block:: bash
+
+    composer require sulu/sulu:"~x.x.x" --no-update
+
+After this all dependencies should be updated with the following command:
```
Comment (niklasnatter) [inline upgrades/upgrade-2.x.rst]:
```suggestion
After this, you can update all dependencies of your project by executing the following command:
```

File: upgrades/upgrade-2.x.rst [hunk 3]
```diff
@@ -5,52 +5,67 @@ This upgrade guide describes how to upgrade a Sulu 2.x project to any newer vers
...
+When upgrading the project, you need to update the build to match the new Sulu version.
+To simplify this step, Sulu provides a command to update the JavaScript build in the project:
 
 .. code-block:: bash
 
     $ bin/console sulu:admin:update-build
 
+.. note::
+
+    For other ways to build the admin interface see :doc:`../cookbook/build-admin-frontend` documentation.
```
Comment (niklasnatter) [inline upgrades/upgrade-2.x.rst]:
```suggestion
    Have a look at the :doc:`../cookbook/build-admin-frontend` documentation if you want to update the 
    JavaScript build in the project without using the ``sulu:admin:update-build`` command.
```

**Review by niklasnatter — APPROVED** [review]

**Comment by alexander-schranz** [comment]
@nnatter changes applied!

**Pull request merged**